### PR TITLE
[FR] osu! wiki Contribution Guide

### PR DESCRIPTION
**Pull request by cl8n**
#2545

### CODE REVIEW

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+### Déplacement de fichiers existants
+
+1\. Utilisez l'explorateur Windows pour déplacer des fichiers.
+
+2\. Continuer à [Committing and pushing](#committing-and-pushing).
+
+### Créer de nouveaux fichiers
+
+*Attention: Si vous envisagez de créer des fichiers d’article à l’aide de l’explorateur Windows, assurez-vous que l'option `Extensions de nom de fichier` est activer.* Voir [Comment afficher ou masquer les extensions de nom de fichier dans l'Explorateur Windows](https://support.microsoft.com/en-us/help/865219/how-to-show-or-hide-file-name-extensions-in-windows-explorer) pour les instructions.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md]:
```suggestion
*Attention: Si vous envisagez de créer des fichiers d’article à l’aide de l’explorateur Windows, assurez-vous que l'option `Extensions de nom de fichier` est activée.* Voir [Comment afficher ou masquer les extensions de nom de fichier dans l'Explorateur Windows](https://support.microsoft.com/en-us/help/865219/how-to-show-or-hide-file-name-extensions-in-windows-explorer) pour les instructions.
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+1\. Utilisez l'explorateur Windows, votre éditeur d'images préféré et/ou votre éditeur de texte préféré pour créer de nouveaux fichiers. Si vous créez de nouveaux articles ou traductions, renommez le fichier pour suivre les instructions. [Paramètres régionaux dans les critères de style d'article](/wiki/ASC/#locales).
+
+2\. Continuer à [Committing and pushing](#committing-and-pushing).
+
+### Suppression de fichiers
+
+1\. Utilisez l'explorateur Windows pour supprimer des fichiers.
+
+2\. ontinuer à [Committing and pushing](#committing-and-pushing).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md]:
```suggestion
2\. Continuez à [Committing and pushing](#committing-and-pushing).
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+4\. Sélectionnez les fichiers à valider.
+
+5\. Entrez le résumé de validation. Celui-ci est limité à 72 caractères et **doit être en anglais**.
+
+6\. Entrez la description. Ceci est facultatif, n’a pas de limite de caractères et peut être dans n’importe quelle langue.
+
+![](img/github-desktop-first-commit.jpg "Premier commit")
+
+7\. Cliquez sur le boutton bleu `Commit to` et, le cas échéant, recommencez à partir de l'étape 4 pour effectuer les autres commits que vous souhaitez effectuer.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md]:
```suggestion
7\. Cliquez sur le bouton bleu `Commit to` et, le cas échéant, recommencez à partir de l'étape 4 pour effectuer les autres commits que vous souhaitez effectuer.
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+![](img/github-desktop-second-commit.jpg "Deuxième commit")
+
+8\. Une fois que vous avez terminé vos commits, cliquez sur le bouton `Publish branch` en haut à droite.
+
+![](img/github-desktop-push.jpg)
+
+9\. Continuer à [Finition](#finishing).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md]:
```suggestion
9\. Continuez à [Finition](#Finition).
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+2\. Accédez au fichier que vous souhaitez modifier. Les articles sont situés dans le dossier `wiki` et sont organisés par les noms de dossier étant les noms d'article, tandis que les fichiers `.md` sont les traductions.
+
+3\. Une fois que vous avez trouvé le bon fichier, cliquez dessus.
+
+4\. Cliquez sur l'icône du crayon.
+
+![](img/online-editing.jpg "En cliquant sur cette icône représentant un crayon dans GitHub, vous accédez à l'éditeur Web.")
+
+5\. Vous pouvez maintenant apporter vos modifications. Lorsque vous effectuez des modifications, essayez de suivre les [Article Critères de style](/wiki/ASC) aussi près que possible.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
5\. Vous pouvez maintenant apporter vos modifications. Lorsque vous effectuez des modifications, essayez de suivre les [Critères de style pour les articles](/wiki/ASC) aussi fidèlement que possible.
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+6\. Continuer à [Saving and committing](#saving-and-committing).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
6\. Continuez à [Saving and committing](#saving-and-committing).
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+3\. Au-dessus de l'éditeur de texte, il y a un chemin pour l'emplacement du fichier, cliquez sur la zone de texte du fichier actuel.
+
+![](img/online-move.jpg)
+
+4\. Pour changer le chemin, appuyez sur `Backspace` et répétez jusqu'à ce que la zone de texte recule d'un niveau. Pour monter le fichier d’un niveau à l’autre, appuyez sur `/`.
+
+5\. Une fois que vous avez atteint le niveau requis, saisissez un autre nom de dossier et/ou le nom de la région (en minuscules) suivi de `.md`.
+
+6\. Continuer à [Saving and committing](#saving-and-committing).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
6\. Continuez à [Saving and committing](#saving-and-committing).
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+1\. Ouvrez l'onglet de code sur votre fork.
+
+2\. Accédez au dossier dans lequel les fichiers doivent être téléchargés.
+
+3\. Cliquez sur `Create new file`.
+
+4\. Entrez l'emplacement et/ou le nom de fichier de l'article que vous créez. Le chemin doit commencer par `wiki/`, suivi du titre de l'article, suivi du nom de l'environnement local (en minuscules) suivi de `.md`. ou une liste de lieux, voir le [Paramètres régionaux dans les critères de style d'article](/wiki/ASC/#locales).
+
+5\. Continuer à [Saving and committing](#saving-and-committing).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
5\. Continuez à [Saving and committing](#saving-and-committing).
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+
+1\. Ouvrez l'onglet de code sur votre fork.
+
+2\. Accédez au dossier dans lequel les fichiers doivent être téléchargés.
+
+3\. Cliquez sur `Upload files`.
+
+4\. Sélectionnez les fichiers que vous souhaitez télécharger.
+
+5\. Continuer à [Saving and committing](#saving-and-committing).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
5\. Continuez à [Saving and committing](#saving-and-committing).
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+
+*Attention: Ceci est **non recommandé** car vous ne pouvez supprimer qu'un fichier à la fois! Cela peut créer plusieurs commits "stupides" qui peuvent amener les relecteurs à fermer ou à caler votre pull request !*
+
+1\. Ouvrez l'onglet de code sur votre fork.
+
+1\. Naviguez jusqu'au fichier que vous souhaitez supprimer et ouvrez-le.
+
+3\. Cliquez sur l'icône de la corbeille.
+
+4\. Continuer à [Saving and committing](#saving-and-committing).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
4\. Continuez à [Saving and committing](#saving-and-committing).
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+
+## Saving and committing
+
+1\. Lorsque vous avez terminé vos modifications, faites défiler vers le bas de la page. Vous pouvez voir quelque chose de semblable à ceci:
+
+![](img/online-commit-changes-empty.jpg "S'il vous plaît ne laissez pas ceci vides!")
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
![](img/online-commit-changes-empty.jpg "S'il vous plaît ne laissez pas ceci vide !")
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+
+3\. Vous pouvez utiliser la description facultative pour être plus détaillée si nécessaire. Un bon commit peut ressembler à quelque chose comme ça :
+
+![](img/online-commit-changes-filled.jpg "C'est ok !")
+
+4\. S'il s'agit de la première modification que vous effectuez pour cet article, sélectionnez l'option `Create a new branch for this commit and start a pull request.`. Cela vous permettra d’apporter à l’avenir des modifications d’articles multiples, mais différentes (passez à l’étape 5). Sinon, sélectionnez `Commit directory to the {xxxxxx} branch.` (et passez à l'étape 6).
+
+5\. Nommez cette branche comme bon vous semble, assurez-vous de bien noter le nom que vous avez choisi.
+
+6\. Cliquez sur le bouton vert. il va lire soit `Propose file change` ou `Commit changes`, Cela dépend si vous créez une nouvelle branche ou non.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
6\. Cliquez sur le bouton vert. il va proposer soit `Propose file change` ou `Commit changes`, cela dépend si vous créez une nouvelle branche ou non.
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+
+7\. Cela fera deux choses : sauvegardez vos modifications dans la branche que vous venez de nommer et cela ouvrira la page de demande de tirage.
+
+![](img/pull-request-pippi-osu--osu-wiki.jpg "Ne fais rien ici! (Ceci est la mauvaise page!)")
+
+8\. Continuer à [Finition](#finishing).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
8\. Continuez à [Finition](#finishing).
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+
+## Finition
+
+Vous pouvez maintenant revenir à  [osu! Guide de contribution du wiki](/wiki/owcg/#finishing) pour la dernière information concernant votre demande et vos modifications.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/GitHub_Web_Interface/fr.md]:
```suggestion
Vous pouvez maintenant revenir au [Guide de contribution du osu!wiki](/wiki/owcg/#finition) pour la dernière information concernant votre demande et vos modifications.
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 15]
```diff
@@ -0,0 +1,108 @@
+# osu! Guide de contribution au wiki
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
# Guide de contribution au osu!wiki
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+
+Merci de votre intérêt pour la fabrication de l'osu! wiki en mieux! Ce guide de démarrage rapide est destiné aux nouveaux arrivants qui n'ont jamais travaillé avec GitHub et osu! Workflow de contribution wiki avant. Si vous avez déjà utilisé GitHub, n'hésitez pas à ignorer ce guide et à suivre le flux de travaux commun aux branches de fonctionnalités pour les mises à jour de contenu dans ce référentiel.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
Merci de votre intérêt pour aider à rendre l'osu! wiki meilleur ! Ce guide de démarrage rapide est destiné aux nouveaux arrivants qui n'ont jamais travaillé avec GitHub et les workflow de contribution d'osu!wiki avant. Si vous avez déjà utilisé GitHub, n'hésitez pas à ignorer ce guide et à suivre le flux de travaux commun aux branches de fonctionnalités pour les mises à jour de contenu dans ce référentiel.
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+
+Si, à un moment quelconque, vous êtes confus et/ou avez besoin d’une aide supplémentaire, n’hésitez pas à envoyer un message au [osu!dev Discord](https://discord.gg/ppy) (channel `#osu-wiki`).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
Si, à un moment quelconque, vous êtes confus et/ou avez besoin d’une aide supplémentaire, n’hésitez pas à envoyer un message au [Discord osu!dev](https://discord.gg/ppy) (canal `#osu-wiki`).
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+
+## Commencer
+
+### Enregistrement
+
+1. [S'inscrire](https://github.com/join) pour un compte GitHub si vous n'en avez pas.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
1. [S'inscrire](https://github.com/join) sur GitHub si vous n'avez pas de compte.
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 19]
```diff
@@ -0,0 +1,108 @@
...
+
+2. [Se connecter](https://github.com/login).
+
+3. Continuer à [Forking](#forking).
+
+### Forking
+
+1. Aller au [`osu-wiki` repo](https://github.com/ppy/osu-wiki).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
1. Allez au [repo `osu-wiki`](https://github.com/ppy/osu-wiki).
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 20]
```diff
@@ -0,0 +1,108 @@
...
+
+![](img/fork.jpg)
+
+3. Cliquez sur `Fork` encore pour aller à votre fork.
+
+4. Voir quelque chose de similaire à l'image ci-dessous signifie que vous avez créé un fork de `ppy/osu-wiki` et que vous êtes sur votre fork.
+
+![](img/forked.jpg)
+
+5. Continuer à [Synchroniser votre fork](#syncing-your-fork).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
5. Continuez à [Synchroniser votre fork](#Synchroniser-votre-fork).
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 21]
```diff
@@ -0,0 +1,108 @@
...
+
+### Synchroniser votre fork
+
+1. Allez à votre fork du `osu-wiki`.
+
+2. Au-dessus des fichiers, il y a du texte pour vous dire si votre branche est "derrière", "en avant" ou "même" avec `ppy:master`. S'il indique que la branche est "en retard" ou "en avance" avec un nombre quelconque de commits en arrière-plan, votre branche est obsolète.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
2. Au-dessus des fichiers, il y a du texte pour vous dire si votre branche est "derrière", "en avant" ou "la même" que `ppy:master`. S'il indique que la branche est "en retard" ou "en avance" avec un nombre quelconque de commits en arrière-plan, votre branche est obsolète.
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 22]
```diff
@@ -0,0 +1,108 @@
...
+
+![](img/fork-even.jpg "OK \(Pas de commits et pas obsolètes\)")
+
+![](img/fork-ahead.jpg "OK \(Ce sont vos commits\)")
+
+![](img/fork-behind.jpg "Potentiellement mauvais \(votre branche est obsolète\)")
+
+![](img/fork-ahead-behind.jpg "Potentiellement mauvais \(votre branche est obsolète avec vos commits\) ")
+
+3. Ce n'est pas vraiment un problème Autrement dit, si vous ne modifiez pas un fichier qui a déjà été modifié, il est peu probable que plus vous êtes en retard.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
3. Ce n'est pas vraiment un problème; autrement dit, si vous ne modifiez pas un fichier qui a déjà été modifié, il est peu probable que vous soyez en retard.
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 23]
```diff
@@ -0,0 +1,108 @@
...
+
+4. Pour résoudre ce problème, voir [Ma branche est obsolète ! section de l'article sur les problèmes courants](/wiki/owcg/Common_Issues/#my-branch-is-out-of-date!).
+
+5. Lorsque vous avez terminé l’étape 4, passez à [Modification en ligne ou localement](#editing-online-or-locally).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
5. Lorsque vous avez terminé l’étape 4, passez à [Modification en ligne ou localement](#Edition-en-ligne-ou-localement).
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 24]
```diff
@@ -0,0 +1,108 @@
...
+
+## Edition en ligne ou localement
+
+À ce stade, vous avez deux choix:
+
+- [Interface Web GitHub](/wiki/owcg/GitHub_Web_Interface) - edit online; this is best for single article edits
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
- [Interface Web GitHub](/wiki/owcg/GitHub_Web_Interface) - éditer en ligne; c'est le meilleur choix pour l'édition d'un seul article
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 25]
```diff
@@ -0,0 +1,108 @@
...
+- [GitHub Desktop](/wiki/owcg/GitHub_Desktop) - éditer localement; ceci est préférable pour les modifications d'article simples et/ou multiples (ceci inclut le téléchargement, la suppression et le déplacement d'images ou de fichiers)
+
+*Remarque: vous n'êtes pas limité aux deux choix énumérés ci-dessus pour l'édition. Pour que ce guide soit simple et concis, ces articles ne discuteront pas d’utilisation d’autres outils en profondeur ou du tout. Il existe d’autres applications tierces qui peuvent faire plus ou moins que ce que font déjà l’Interface Web GitHub et GitHub Desktop.*
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
*Remarque: vous n'êtes pas limité aux deux choix énumérés ci-dessus pour l'édition. Pour que ce guide soit simple et concis, ces articles ne parleront pas d’utilisation d’autres outils en profondeur ou pas du tout. Il existe d’autres applications tierces qui peuvent faire plus ou moins que ce que font déjà l’Interface Web GitHub et GitHub Desktop.*
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 26]
```diff
@@ -0,0 +1,108 @@
...
+
+**Lorsque vous avez terminé *l'interface Web GitHub* ou *GitHub Desktop*, vous pouvez passer à la section suivante.**
+
+## Finition
+
+### Ouvrir un Pull Request
+
+1. Aller au [`ppy/osu-wiki` repo](https://github.com/ppy/osu-wiki).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
1. Allez au [repo `ppy/osu-wiki`](https://github.com/ppy/osu-wiki).
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 27]
```diff
@@ -0,0 +1,108 @@
...
+
+1. Cliquez sur la liste déroulante `compare` et sélectionnez celle avec la branche que vous avez créée (elles sont classées par ordre alphabétique).
+
+![](img/compare-branch.jpg)
+
+7. Cliquez sur `Create pull request`.
+
+8. Entrez le titre en anglais. Cela devrait être une très brève explication de ce que vous avez changé.
+
+  Pour les traductions d'articles,incluez le nom de langue en deux lettres de vos traductions entre parenthèses avant le titre. Votre titre peut simplement être le nom du ou des articles que vous traduisez. Par exemple, `[FR] BBCode` indiquerait que vous mettez à jour la traduction française du [BBCode article](/wiki/BBCode).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
  Pour les traductions d'articles,incluez le nom de langue en deux lettres de vos traductions entre parenthèses avant le titre. Votre titre peut simplement être le nom du ou des articles que vous traduisez. Par exemple, `[FR] BBCode` indiquerait que vous mettez à jour la traduction française de l'[article BBCode](/wiki/BBCode).
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 28]
```diff
@@ -0,0 +1,108 @@
...
+
+9. Remplissez la zone de description avec un résumé de vos modifications. Vous devez mentionner toute information pertinente à votre demande d'extraction, telle que son statut d'achèvement et tout ce que vous souhaitez que les réviseurs sachent. Vous pouvez également faire en sorte que votre demande d'extraction ferme automatiquement les problèmes de fusion en écrivant "resolves #1" ou "closes #1", dans la description. (voir [Fermer des issues en utilisant des mots clés](https://help.github.com/articles/closing-issues-using-keywords/) sur l'aide de GitHub).
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
9. Remplissez la zone de description avec un résumé de vos modifications. Vous devez mentionner toute informations pertinentes à votre demande d'extraction, telle que son statut d'achèvement et tout ce que vous souhaitez que les réviseurs sachent. Vous pouvez également faire en sorte que votre demande d'extraction ferme automatiquement les problèmes de fusion en écrivant "resolves #1" ou "closes #1", dans la description. (voir [Fermer des issues en utilisant des mots clés](https://help.github.com/articles/closing-issues-using-keywords/) sur l'aide de GitHub).
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 29]
```diff
@@ -0,0 +1,108 @@
...
+
+10. Une fois que vous êtes prêt, cliquez sur `Create pull request`.
+
+![](img/new-pull-request.png)
+
+11. Voir [Reviews](#reviews) et [Merging](#merging) en dessous.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
11. Voir [Commentaires](#commentaires) et [Fusion](#fusions) en dessous.
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 30]
```diff
@@ -0,0 +1,108 @@
...
+
+### Commentaires
+
+Une fois que vous avez créé votre pull request, les autres osu! Les éditeurs de wiki peuvent revoir vos modifications pour vous aider à détecter certaines erreurs que vous avez peut-être manquées. **Vous devrez suivre ces avis,** sinon votre pull request pourrait être marquée pour fermeture ! Si vous souhaitez que quelqu'un vérifie votre demande d'extraction, vous pouvez demander à un autre osu! éditeurs de wiki dans le [osu!dev Discord](https://discord.gg/ppy) (`#osu-wiki` channel) ou dans les commentaires GitHub.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
Une fois que vous avez créé votre pull request, les autres éditeurs de l'osu!wiki peuvent voir vos modifications pour vous aider à détecter certaines erreurs que vous avez peut-être manquées. **Vous devrez suivre ces avis,** sinon votre pull request pourrait être marquée pour fermeture ! Si vous souhaitez que quelqu'un vérifie votre demande d'extraction, vous pouvez demander à un autre éditeur d'osu!wiki dans le [Discord osu!dev](https://discord.gg/ppy) (canal `#osu-wiki`) ou dans les commentaires GitHub.
```

File: wiki/osu!_wiki_Contribution_Guide/fr.md [hunk 31]
```diff
@@ -0,0 +1,108 @@
...
+
+### Fusions
+
+Pour que vos modifications apparaissent en direct sur osu! wiki, votre pull request doit être fusionnée. Une fois que votre pull request a été examinée et approuvée, vous pouvez utiliser la section commentaires de GitHub pour demander à quelqu'un de la fusionner ou faire la même chose dans le menu déroulant. [osu!dev Discord](https://discord.gg/ppy) (`#osu-wiki` channel). Une fois la fusion effectuée, vos modifications prendront jusqu'à cinq heures pour apparaître en direct sur osu! wiki.
```
Comment (almaloste) [inline wiki/osu!_wiki_Contribution_Guide/fr.md]:
```suggestion
Pour que vos modifications apparaissent en direct sur osu!wiki, votre pull request doit être fusionnée. Une fois que votre pull request a été examinée et approuvée, vous pouvez utiliser la section commentaires de GitHub pour demander à quelqu'un de la fusionner ou faire la même chose dans le menu déroulant. [Discord osu!dev](https://discord.gg/ppy) (canal `#osu-wiki`). Une fois la fusion effectuée, vos modifications prendront jusqu'à cinq heures pour apparaître en direct sur osu!wiki.
```

File: wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md [hunk 32]
```diff
@@ -0,0 +1,103 @@
...
+
+*Remarque: Ceci est techniquement optionnel, mais fortement recommandé pour des raisons que vous devrez rechercher vous-même.*
+
+1\. Clique sur le menu déroulant  `Current branch`. Ceci est situé en haut du menu.
+
+2\. Entrez un nom pour cette branche. Il est suggéré d'utiliser le nom de l'article que vous modifiez en tant que nom de branche suivi d'un numéro. Par exemple, éditer le `Forums` article peut entraîner la branche étant appelée `forums1`.
+
+![](img/github-desktop-branch.jpg)
+
+3\. Continuer à [Édition](#editing).
```
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md]: the anchors are lowercased on the site, but I'm not sure if this accented "E" becomes lowercase, i'll have to check

[the remark this answers is not in the document]
Comment (Joehuu) [reply wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/fr.md]: Check it on github, it does become a lowercase accented e.

**Pull request merged**